3-platform/order/domain/pom.xml

```diff
@@ -39,6 +39,12 @@
       <artifactId>mockito-junit-jupiter</artifactId>
       <version>5.2.0</version>
     </dependency>
+
+    <dependency>
+      <groupId>com._6core.lib.java</groupId>
+      <artifactId>reactive-postges-persistent</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
```

Comment:
register to the 6core_bom to get rid of version
move to the infra layer (remember that domain/usecase layers should know nothing about frameworks, databases, communication protocols that we are using to replace it simply)
```suggestion
```